rust/signer/src/lib.rs

```diff
@@ -341,4 +342,31 @@ mod tests {
 
         assert!(is_valid);
     }
+
+    #[test]
+    fn test_substrate_sign_with_ref() {
+        let msg: String = b"Build The Future".to_hex();
+        let data_pointer = decrypt_data_ref(ENCRYPTED_SEED, String::from(PIN)).unwrap();
+        let signature_by_ref = substrate_brainwallet_sign_with_ref(data_pointer, &msg).unwrap();
+        let is_valid = schnorrkel_verify(SURI, &msg, &signature_by_ref).unwrap();
+        assert!(is_valid);
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
This flag does not prevent `malloc` error on test, still make the test failed.

Reply:
I don't think we should test this.  It is testing a pattern that should never occur, ie. accessing dropped memory.